package/src/components/Indicators/LoadingErrorIndicator.tsx

```diff
@@ -48,7 +48,7 @@ const LoadingErrorWrapper: React.FC<LoadingErrorWrapperProps> = (props) => {
 };
 
 export type LoadingErrorProps = {
-  error?: boolean;
+  error?: unknown;
```

Comment:
Why are we using unknown? I would thing we could make a more useful type for the errors? I would think at least the built-in `Error` type makes sense, or [the axios error type](https://github.com/axios/axios/issues/3612#issuecomment-770224236) if we want to be more specific (which I doubt)

Reply:
Do we use Axios here, I suppose Axios provides its own types which are confined for their own project and response. I have pretty much used Axios, express, etc. in the past, and that is what I know about it, not sure, I could be wrong.
Note: This is the exact reason the client, backend should introduce its own error types.

Reply:
@khushal87 I think Mads has a point here. We can simply add error type from axios as well - `AxiosError | Error | boolean`.

Reply:
@khushal87 we can add `@types/axios` as a devDependency, which won't affect our built product at all.

Could also alias the union Vish added above to not have to repeat it everywhere, like

`type StreamChatError = AxiosError | Error | boolean` for example, if that seems like less stuff to change around. (Or a more specific name if you can think of one of course)

Reply:
Yeah go for it. The only thing is, I wouldn't name it `StreamChatError`. This seems like something js client should expose and if we introduce in RN sdk, we loose the opportunity for js client to introduce this type. I would just keep it `ErrorType`